src/main/java/com/tastetracker/domain/review/ReviewDtoMapper.java

```diff
@@ -22,4 +23,17 @@ static ReviewDto map( Review review, List<Rating> ratings){
             ratingService
         );
     }
+
+    static ReviewDto mapApiResponse( Review review, Rating rating)
```

Comment:
zmieniłbym nazwę metody, na po prostu map, w argumentach podajemy co innego niż w pierwszej metodzie, to co za różnica?